packages/framework/src/Migrations/Version20241128111224.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20241128111224 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE orders ADD free_transport_and_payment_applied BOOLEAN DEFAULT NULL');
+        $this->sql('ALTER TABLE orders ALTER free_transport_and_payment_applied DROP DEFAULT');
```

Comment:
I think that adding this to order is not correct way - I think, that order should always look into its promo codes and look, if there is one for free transport and payment instead. 

One of the reasons is, that you do not change this to false, if you remove free transport and payment from order e.g. in administration.

Reply:
after discussion, removed the nullability in 44b0039ec9